### PR TITLE
Disable files under review when addon disabling

### LINKED ISSUE #1107: Disable all files pending review when a developer disables their add-on
When a developer decides to disable their add-on while a file is pending review, we shall disable the file to prevent review.
If the file is being reviewed while the add-on is disabled, the file will not be signed (even if the file was approved).

Example where this happened:
https://addons.mozilla.org/en-US/developers/feed/my-erp

### PR DESCRIPTION
**Pull request by eviljeff**
fixes #1107

### CODE REVIEW

File: src/olympia/devhub/views.py [hunk 1]
```diff
@@ -392,6 +392,9 @@ def cancel(request, addon_id, addon):
 @post_required
 def disable(request, addon_id, addon):
     addon.update(disabled_by_user=True)
+    if (addon.latest_version):
```
Comment (EnTeQuAk) [inline src/olympia/devhub/views.py]: I don't think you need the parens here.

Comment (eviljeff) [inline src/olympia/devhub/views.py]: the curse of javascript

**Comment by EnTeQuAk** [comment]
@eviljeff can you fix the one small nitpick regarding parens in the if? Otherwise, r+wc

I don't know that part of the code good enough but do we allow enabling addons again? If so, we would need to enable the files then again too.

**Comment by eviljeff** [comment]
In the comments in #1107 Jorge says the file shouldn't be re-enabled, even after the add-on is unhidden.

**Comment by EnTeQuAk** [comment]
Then nevermind :)

**Pull request merged**